src/test/java/com/example/HelloWorldResourceTest.java

```diff
@@ -19,41 +27,81 @@ public class HelloWorldResourceTest {
             ResourceHelpers.resourceFilePath("config/test-it.yml")
     );
 
-//    private static WireMockServer wireMockServer;
+    private static WireMockServer wireMockServer;
 
-//    @BeforeClass
-//    public static void setUp() {
-//        // Start WireMock server on port 8080
-//        wireMockServer = new WireMockServer(8080);
-//        wireMockServer.start();
-//    }
-
-    @Test
-    void ShouldReturnHelloWorldText() {
-        WireMockServer wireMockServer = new WireMockServer(Integer.parseInt(extension.getConfiguration().getNodePort()));
+    @BeforeAll
+    public static void setUp() {
+        wireMockServer = new WireMockServer(Integer.parseInt(extension.getConfiguration().getNodePort()));
         wireMockServer.start();
-        // Stub the API response using WireMock
         configureFor("localhost", Integer.parseInt(extension.getConfiguration().getNodePort()));
-        stubFor(get((urlEqualTo("/")))
+    }
+
+    @AfterAll
+    public static void tearDown() {
+        wireMockServer.stop();
+    }
+
+    @Test
+    void shouldReturnHelloWorldGreeting() {
+
+        String getResponseJson = """
+                {
+                    "text": "Hello, World!"
+                }
+                """;
+
+        stubFor(get((urlEqualTo("/greeting")))
                 .willReturn(aResponse()
                         .withStatus(200)
                         .withHeader("Content-Type", "application/json")
-                        .withBody("{\"text\": \"Hello, world!\"}")));
+                        .withBody(getResponseJson)));
 
-        // Send a request to the API using REST Assured
-            given()
+        given()
                 .port(extension.getLocalPort())
                 .when()
-                .get("/")
+                .get("/greeting")
                 .then()
                 .statusCode(200)
                 .contentType("application/json")
-                .body("text", is("Hello, world!"));
-        wireMockServer.stop();
+                .body("text", is("Hello, World!"));
+
     }
 
-//    @AfterClass
-//    public static void setDown() {
-//        wireMockServer.stop();
-//    }
+    @Test
+    void shouldProcessAndReturnName() {
+
+        String postRequestJson = """
+                {
+                    "name": "Hello from Ali!"
+                }
+                """;
+
+        String postResponseJson = """
+                {
+                    "name": "Hello from Ali!"
```

Comment:
Can you change the JSON as it's identical to the one above? E.g. have the node endpoint return it capitalized etc...